### PR TITLE
initial commit for pyrsm 0.5.7

### PR DESCRIPTION
**Pull request by vnijs**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Package maintainer `vnijs` confirming that package will be maintained

### CODE REVIEW

**Comment by vnijs** [comment]
Just checking if there is anything else I need to do before this can be evaluated. Thanks

**Comment by vnijs** [comment]
@conda-forge-admin, please ping team

**Review by BastianZim — CHANGES_REQUESTED** [review]

File: recipes/pyrsm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/pyrsm/
+  summary: Python functions for Customer Analytics at the Rady School of Management (RSM)
+  dev_url: https://github.com/vnijs/pyrsm
+  license: GPL-2.0-or-later
```
[the remark this answers is not in the document]
Comment (vnijs) [reply recipes/pyrsm/meta.yaml:47]: @vnijs I would prefer to use AGPL 3but that wasn't listed as an option. Can I use AGPL3 for conda-forge?

[the remark this answers is not in the document]
Comment (BastianZim) [reply recipes/pyrsm/meta.yaml:47]: Yes, just use the one that I suggested. That is GNU Affero General Public License v3.0. 

By the way, a full list can be found here: https://spdx.org/licenses/

**Review by BastianZim — APPROVED** [review]

**Pull request merged**